src/content/reference/okteto-manifest.mdx

```diff
@@ -252,13 +254,38 @@ Your deploy commands will be executed before deploying your Docker Compose files
 
 #### Divert
 
-Divert is useful when you have many services and you only want to deploy a subset of them in your development environment, while connecting these services with a shared environment that runs all your microservices.
+Divert enables you to create development environments with only the services that you are actively developing, while being able to share an existing shared environment that runs all your microservices. For large environments, this often results in a substantial reduction of infrastructure cost and complexity.
 
+<Tabs
+  defaultValue="default"
+  values={[
+    { label: 'Default', value: 'default', },
```

Comment:
The default value for `driver` is `nginx` because the default implementation is based in the nginx ingress controller

Reply:
Not sure I get your comment here. Are you suggesting that we explicitly include "driver: nginx" on the configuration doc?

Reply:
yes, the default `driver` is `nginx`

Reply:
Updated here: https://github.com/okteto/docs/pull/1006